Core/src/RequestWrapperTrait.php

```diff
@@ -178,11 +178,17 @@ public function getCredentialsFetcher()
             }
         }
 
-        return new FetchAuthTokenCache(
-            $fetcher,
-            $this->authCacheOptions,
-            $this->authCache
-        );
+        if ($fetcher instanceof FetchAuthTokenCache::class) {
```

Comment:
This just needs to be `FetchAuthTokenCache` :)

Reply:
Oops, sorry! PTAL :-)